dnf-behave-tests/dnf/plugins-core/enable-plugins-dashes.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Tests for enabling plugins containing dashes in names
+
+Background: Enable plugins
+  Given I do not disable plugins
+    And I use repository "dnf-ci-fedora"
+
+@bz1980712
+Scenario: Existing plugin containing dashes in the name could be disabled
+   When I execute dnf with args "repoquery empty --disableplugin=config-manager --verbose"
+   Then the exit code is 0
+    And stdout does not contain "config-manager"
+    And stderr does not contain "No matches found for the following disable plugin patterns: config-manager"
+
+@bz1980712
+Scenario: Existing plugin containing dashes in the name could be enabled
+   When I execute dnf with args "--disableplugin=config-manager"
```

Comment:
`--disableplugin` only affects the current run of dnf, disablement of plugin is not persistent. That said, this line doesn't do anything useful and does not affect the following `repoquery --enableplugin...`
To simulate enabling a disabled plugin, you coud use `--disableplugin='*' --enableplugin='config-manager`

Reply:
Thank you Marek, how could I've missed that :slightly_smiling_face: